README.md

```diff
@@ -6,6 +6,13 @@ This is an OpenWRT feed with a Linux kernel module implementing flexible NAT46.
 Compiling
 =========
 
+The module by default uses procfs for communication between the user and kernel space.
+To use Netlink sockets instead, add the following to nat46/nat46/modules/Makefile
+when compiling:
+```
+EXTRA_CFLAGS += -DPROTO_NETLINK
```

Comment:
The define should be a bit more specific for NAT46 module and more self-descriptive. NAT46_USE_NETLINK_CONFIG or something like that.

Reply:
Thanks for your feedback. I'll keep your suggestion in mind as I update my patch.